boards.txt

```diff
@@ -2780,6 +2780,479 @@ aventen_s3_sync.menu.EraseFlash.all.upload.erase_cmd=-e
+BharatPi-Node-Wifi.build.flash_freq=40m
+BharatPi-Node-Wifi.build.flash_mode=dio
+BharatPi-Node-Wifi.build.boot=dio
+BharatPi-Node-Wifi.build.partitions=default
+BharatPi-Node-Wifi.build.defines=
+BharatPi-Node-Wifi.build.loop_core=
+BharatPi-Node-Wifi.build.event_core=
+
+BharatPi-Node-Wifi.menu.PartitionScheme.default=Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.default.build.partitions=default
+BharatPi-Node-Wifi.menu.PartitionScheme.defaultffat=Default 4MB with ffat (1.2MB APP/1.5MB FATFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.defaultffat.build.partitions=default_ffat
+BharatPi-Node-Wifi.menu.PartitionScheme.default_8MB=8M with spiffs (3MB APP/1.5MB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.default_8MB.build.partitions=default_8MB
+BharatPi-Node-Wifi.menu.PartitionScheme.default_8MB.upload.maximum_size=3342336
+BharatPi-Node-Wifi.menu.PartitionScheme.minimal=Minimal (1.3MB APP/700KB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.minimal.build.partitions=minimal
+BharatPi-Node-Wifi.menu.PartitionScheme.no_ota=No OTA (2MB APP/2MB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.no_ota.build.partitions=no_ota
+BharatPi-Node-Wifi.menu.PartitionScheme.no_ota.upload.maximum_size=2097152
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3g=No OTA (1MB APP/3MB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3g.build.partitions=noota_3g
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3g.upload.maximum_size=1048576
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_ffat=No OTA (2MB APP/2MB FATFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_ffat.build.partitions=noota_ffat
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_ffat.upload.maximum_size=2097152
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3gffat=No OTA (1MB APP/3MB FATFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3gffat.build.partitions=noota_3gffat
+BharatPi-Node-Wifi.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+BharatPi-Node-Wifi.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.huge_app.build.partitions=huge_app
+BharatPi-Node-Wifi.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+BharatPi-Node-Wifi.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+BharatPi-Node-Wifi.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+BharatPi-Node-Wifi.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.fatflash.build.partitions=ffat
+BharatPi-Node-Wifi.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+BharatPi-Node-Wifi.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
+BharatPi-Node-Wifi.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+BharatPi-Node-Wifi.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
```

Comment:
Also remove those partitions -> needs 16MB of Flash. Apply to all boards please.
```suggestion
```